RetailCoder.VBE/UI/Command/FindAllReferencesCommand.cs

```diff
@@ -81,7 +80,7 @@ private SearchResultsViewModel CreateViewModel(Declaration declaration)
                 new SearchResultItem(
                     reference.ParentNonScoping,
                     new NavigateCodeEventArgs(reference.QualifiedModuleName, reference.Selection), 
-                    reference.QualifiedModuleName.Component.CodeModule.get_Lines(reference.Selection.StartLine, 1).Trim()));
+                    reference.QualifiedModuleName.Component.CodeModule.Lines[reference.Selection.StartLine, 1].Trim()));
```

Comment:
huh, no issues with the indexed property? that kept throwing a `COMException` in earlier versions, that's why I've been using `get_Lines` ever since...